vaccine_feed_ingest/runners/ri/arcgis/normalize.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+
+from schema import schema  # noqa: E402
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("ri/arcgis/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["OBJECTID"]
+
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    site = "arcgis"
+    runner = "ri"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "da57e8c8663048a2a9893c636fef63d0"
+    layer = 0
+
+    return f"{runner}:{site}:{arcgis}_{layer}:{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines_field = site["attributes"]["USER_VaxType"].lower().split(",")
+
+    potentials = {
```

Comment:
Could you update these vaccine types to be the ones specified in the wiki: https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema

Sorry, pretty sure these were updated after the previous review 😅

Reply:
Done!

Reply:
Sorry about that - we updated the schema last night to match another system, and I forgot to come back to this PR to share the news. Thank you for the fix!